docs/hosting/configuration/environment-variables/binary-data.md

```diff
@@ -11,10 +11,14 @@ hide:
 
 # Binary data environment variables
 
+/// note | File-based configuration
```

Comment:
@ayatnkw probably worth making this into a snippet as we're reusing it in multiple places. To do this:

1. Create a file in `_snippets` (in the relevant subfolder)
2. Add the snippet everywhere you want to use the contents:

```
--8<-- "_snippets/<my-folder>/<my-file.md>"

Reply:
moved the snippet to self-hosting directory @StarfallProjects